signwriting/visualizer/visualize.py

```diff
@@ -1,6 +1,8 @@
+# pylint: disable=unnecessary-lambda-assignment
```

Comment:
ideally, this should be near the affected line, otherwise in the pyproject

Reply:
there were 4 lines using `lambda` i felt it is redundant to use it 4 time, so what do you suggest i do? in the pyproject config?

Reply:
either project config or next to each line

Reply:
done